udf/bool_to_bytes.sql

```diff
@@ -0,0 +1,12 @@
+/* Convert a boolean to byte array */
+
+CREATE TEMP FUNCTION
+  udf_bool_to_bytes(val BOOLEAN) AS (
+    IF(COALESCE(val, FALSE), udf_zeroed_364_days_active_1_bytes(), udf_zeroed_365_days_bytes()));
```

Comment:
the `COALESCE` here has no effect, because `IF` treats `FALSE` and `NULL` the same:
```suggestion
    IF(val, udf_zeroed_364_days_active_1_bytes(), udf_zeroed_365_days_bytes()));
```